openwisp_users/admin.py

```diff
@@ -370,6 +370,8 @@ def has_delete_permission(self, request, obj=None):
                 return True
             else:
                 return False
+        if request.user.is_superuser:
+            return True
```

Comment:
IMO the logic is clearer if we do:

```
if request.user.is_superuser:
    return True
if obj:
    ...
```

Reply:
That's true @PabloCastellano thanks for the review